docs/docs/building-modules/09-module-interfaces-continued.md

```diff
@@ -0,0 +1,158 @@
+# Module Interfaces Continued
```

Comment:
It would simply call it `AutoCLI` (and the file too), so it shows nice in the menu and it is clearer what it is about.

Reply:
Can you add the following at the top of the file?

```sh
---
sidebar_position: 1
---
```

Reply:
perf, i was thinking this originally but didnt want to change the flow of the current docs